src/Admin/SiteHealth.php

```diff
@@ -157,10 +246,15 @@ private function get_persistent_object_cache_learn_more_action() {
 	 */
 	public function persistent_object_cache() {
 		$is_using_object_cache = wp_using_ext_object_cache();
+		$has_page_caching      = get_transient( self::HAS_PAGE_CACHING_TRANSIENT_KEY );
 
-		$description = '<p>' . __( 'The AMP plugin performs at its best when persistent object cache is enabled. Object caching is used to more effectively store image dimensions and parsed CSS.', 'amp' ) . '</p>';
+		$description = '<p>' . esc_html__( 'The AMP plugin performs at its best when persistent object cache is enabled. Persistent object caching is used to more effectively store image dimensions and parsed CSS using a caching backend rather than using the options table in the database.', 'amp' ) . '</p>';
 
 		if ( ! $is_using_object_cache ) {
+			if ( $has_page_caching ) {
```

Comment:
Given that the page cache test is async and this is direct, there would be a chance of this being a false positive right?

Reply:
Yes, the first time the Site Health tests then this will always be `false`. I don't see a way around that so I think it's acceptable for the first time.

Reply:
Hmm yea, seems like an acceptable trade off.